hr_expense_cancel/readme/INSTALL.rst

```diff
@@ -1,5 +1,2 @@
 It's assumed that for each expense sheet, the *payments* that are reconciled
-with their *journal entry* were created and reconciled by the *Expense Tracker*
-module automatically. Therefore, when this module is installed, these payments
-will automatically be associated with said expense sheet so that they can be
-deleted in case the expense sheet is canceled.
+with their *journal entry* were created and reconciled by the *Expense Tracker* module automatically. Therefore, when this module is installed, these payments will automatically be associated with said expense sheet so that they can be deleted in case the expense sheet is canceled.
```

Comment:
And this? The change is incorrect at RST syntax level

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)